### PR TITLE
feat: added api endpoint

### PR DESCRIPTION
**Pull request by SimonLTheodo**
https://trello.com/c/6UZ8tDfK/67-8-aadev-i-can-get-the-response-from-the-chat-bot-via-an-api-call-to-the-site

### CODE REVIEW

File: src/app/api/route.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import path from 'path';
+dotenv.config({ path: path.resolve(__dirname, '../..', '.env.local') });
+
+export async function POST(request: Request) {
+  const requestBody = await request.json();
+  if (!requestBody.question) {
+    return NextResponse.json('Error: No question provided.', { status: 400 });
+  }
+  const response = await callOpenai(requestBody.question);
+  return NextResponse.json({ response });
```
Comment (matt-newhall) [inline src/app/api/route.ts:13]: is `status: 200` implicit here?

File: src/app/api/route.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { callOpenai } from '@/utils/openai';
+import { NextResponse } from 'next/server';
+import dotenv from 'dotenv';
+import path from 'path';
+dotenv.config({ path: path.resolve(__dirname, '../..', '.env.local') });
```
Comment (matt-newhall) [inline src/app/api/route.ts:5]: isn't this `../../../`? need to go up three directories to hit `.env.local`

File: src/app/api/route.ts (hunk 1 above)
Comment (SimonLTheodo) [reply src/app/api/route.ts:13]: Yep!

File: src/app/api/route.ts (hunk 2 above)
Comment (SimonLTheodo) [reply src/app/api/route.ts]: This is actually unused, so I removed this!

**Review by matt-newhall — APPROVED** [review]

**Pull request merged**